packages/python-django-storages/python-django-storages.spec

```diff
@@ -32,7 +32,39 @@ Obsoletes:      python3.11-%{pypi_name} < %{version}-%{release}
 %description
 %{summary}
 
+%package -n python%{python3_pkgversion}-%{pypi_name}+s3
+Summary: Metapackage for python%{python3_pkgversion}-pulpcore: s3 compatible extra
+Requires: python%{python3_pkgversion}-boto3 >= 1.4.4
 
+%description -n python%{python3_pkgversion}-%{pypi_name}+s3
+This is a metapackage bringing in filecache extra requires for python%{python3_pkgversion}-%{pypi_name}
+It contains no code, just makes sure the dependencies are installed.
+
+%files -n python%{python3_pkgversion}-%{pypi_name}+s3
+%ghost %{python3_sitelib}/%{src_name}-%{version}.dist-info/
+
+%package -n python%{python3_pkgversion}-%{pypi_name}+azure
+Summary: Metapackage for python%{python3_pkgversion}-pulpcore: azure compatible extra
+Requires: python%{python3_pkgversion}-azure-storage-blob >= 12.0.0
+Requires: python%{python3_pkgversion}-azure-core >= 1.13
+
+%description -n python%{python3_pkgversion}-%{pypi_name}+azure
+This is a metapackage bringing in filecache extra requires for python%{python3_pkgversion}-%{pypi_name}
+It contains no code, just makes sure the dependencies are installed.
+
+%files -n python%{python3_pkgversion}-%{pypi_name}+azure
+%ghost %{python3_sitelib}/%{src_name}-%{version}.dist-info/
+
+%package -n python%{python3_pkgversion}-%{pypi_name}+google
+Summary: Metapackage for python%{python3_pkgversion}-pulpcore: google compatible extra
+Requires: python%{python3_pkgversion}-google-cloud-storage>=1.36.1
```

Comment:
We don't package this:
```
[2025-04-08T12:44:24.614Z]         package: python3.12-django-storages+google-1.14.5-4.el9.noarch from repo0
[2025-04-08T12:44:24.614Z]           unresolved deps (2):
[2025-04-08T12:44:24.614Z]             python3.12dist(google-cloud-storage) >= 1.32
[2025-04-08T12:44:24.614Z]             python3.12-google-cloud-storage>=1.36.1
```

Reply:
It's being rebuilt here, I forgot :sweat_smile: 

https://github.com/theforeman/pulpcore-packaging/pull/1849